app/views/view-licences/licence.njk

```diff
@@ -0,0 +1,52 @@
+{% extends 'layout.njk' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% block breadcrumbs %}
+  {{
+    govukBackLink({
+      text: 'Back',
+      href: "/licences"
+    })
+  }}
+{% endblock %}
+
+{% block content %}
+  <h1 class="govuk-heading-l">Licence Number {{ licenceRef }}</h1>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <div class="govuk-tabs" data-module="govuk-tabs">
```

Comment:
This is more just a pointer to have a discussion. I just assumed I'd see the Nunjucks version of the [GOV.UK tabs implementation](https://design-system.service.gov.uk/components/tabs/).

I can see this is the HTML version. We've currently preferred going with Nunjucks over HTML. So, I just wanted to check there wasn't something I was missing or a blocker to doing so in this case.